manual/io/index.md

```diff
@@ -56,784 +46,433 @@ root [1] TFile f1("my.root");
-
-### Streaming a TClonesArray
-
-When writing a {% include ref class="TClonesArray" %}, it bypasses by default the `Streamer` of the member class and uses a more efficient internal mechanism to write the members to the file.
-
-You can override the default and specify that the member class `Streamer` is used by setting the [TClonesArray::BypassStreamer](https://root.cern/doc/master/classTClonesArray.html#a28b32cc35a81a3feac2ee38fe491f47d){:target="_blank"} bit to false:
-
-{% highlight C++ %}
-   TClonesArray *fTracks;
-   fTracks->BypassStreamer(kFALSE);             // Use the member Streamer.
-{% endhighlight %}
-
-
-When the `kBypassStreamer` bit is set, the automatically generated `Streamer` can call directly the [TClass::WriteBuffer](https://root.cern/doc/master/classTClass.html#aff0609831684cbd2162ede5e4cbc3ee7){:target="_blank"} method. Bypassing the `Streamer` improves the performance when writing or reading the objects in the {% include ref class="TClonesArray" %}.
-
-However, the drawback is when a {% include ref class="TClonesArray" %} is written with `split=0` bypassing the `Streamer`, the `StreamerInfo` of the class in the array being optimized, one cannot later use the {% include ref class="TClonesArray" %}. with `split > 0`.
+> **Note**
+> A class with a custom streamer cannot be split, and its members cannot be stored member-wise.
 
-For example, there is a problem with the following scenario: a class `Foo` has a `TClonesArray` of `Bar` objects the `Foo` object is written with `split=0` to tree `T1`. In this case the `StreamerInfo` for the class `Bar` is created in optimized mode in such a way that data members of the same type are written as an array improving the I/O performance. In a new program, `T1` is read and a new tree `T2` is created with the object `Foo` in `split > 1`.
+### Disable storage of TObject data members in derived classes
 
-When the `T2branch` is created, the `StreamerInfo` for the class `Bar` is created with no optimization (mandatory for the split mode). The optimized `Bar` `StreamerInfo` is going to be used to read the {% include ref class="TClonesArray" %} in `T1`. The result are `Bar` objects with data member values not in the right sequence. The solution to this problem is to call `BypassStreamer(kFALSE)` for the {% include ref class="TClonesArray" %}. In this case, the normal `Bar::Streamer` function is  called. The `Bar::Streamer` function works independently if the `Bar` `StreamerInfo` has been generated in optimized mode or not.
+Types do not have to inherit from {% include ref class="TObject" %} for ROOT to be able to read/write them: the presence of a dictionary is sufficient.
 
-## Pointers and references in persistency
+Classes that do inherit from `TObject` can exclude TObject's data members from their I/O invoking `MyClass->Class::IgnoreObjectStreamer()`.
```

Comment:
I think this should be:
```suggestion
Classes that do inherit from `TObject` can exclude TObject's data members from their I/O invoking `MyClass->Class()->IgnoreTObjectStreamer()`.
```

Reply:
You are right, and the instance should not be called like a type:

```suggestion
Classes that do inherit from `TObject` can exclude TObject's data members from their I/O invoking `myObject->Class()->IgnoreObjectStreamer()`.
```